apps/docs/src/styles/_callout.scss

```
@@ -0,0 +1,127 @@
+/* stylelint-disable scss/at-extend-no-missing-placeholder */
+
+/**
+    Usage:
+
+    ```html
+        <div class="kbq-callout kbq-callout_theme">
+            <div class="kbq-callout__header">...</div>
+            <div class="kbq-callout__content">...</div>
+        </div>
+    ```
+*/
+
+@use '@koobiq/icons/fonts/kbq-icons';
+
+%kbq-callout_contrast {
+    border-color: var(--kbq-line-contrast-less);
+    background-color: var(--kbq-background-bg-secondary);
+
+    .kbq-callout__header {
+        color: var(--kbq-foreground-contrast-secondary);
+
+        &::before {
+            @extend .kbq;
+            @extend .kbq-question-circle_16;
+        }
+    }
+
+    .kbq-callout__content {
+        color: var(--kbq-foreground-contrast);
+        background-color: var(--kbq-background-bg);
+    }
+}
+
+.kbq-callout {
+    overflow: hidden;
+    border-width: 2px;
+    border-style: solid;
+    border-radius: var(--kbq-size-s);
+    margin-bottom: var(--kbq-size-xl);
+
+    .kbq-callout__header {
+        display: flex;
+        align-items: center;
+        gap: var(--kbq-size-s);
+        font-size: var(--kbq-md-typography-md-h5-font-size);
+        font-weight: var(--kbq-typography-text-normal-strong-font-weight);
+        line-height: var(--kbq-typography-text-normal-strong-line-height);
+        letter-spacing: var(--kbq-typography-text-normal-strong-letter-spacing);
+    }
+
+    .kbq-callout__content {
+        font-size: var(--kbq-md-typography-md-body-font-size);
+        font-weight: var(--kbq-md-typography-md-body-font-weight);
+        line-height: var(--kbq-md-typography-md-body-line-height);
+        letter-spacing: var(--kbq-md-typography-md-body-letter-spacing);
+        border-radius: var(--kbq-size-s) var(--kbq-size-s) 0;
```

команда  UX завела глобальный токен border-radius, у него такое же значение, но он больше подходит для описания бордер радиуса
```suggestion
        border-radius: var(--kbq-size-border-radius) var(--kbq-size-border-radius) 0;
```